server/etcdmain/help.go

```diff
@@ -237,7 +237,7 @@ Experimental distributed tracing:
   --experimental-distributed-tracing-sampling-rate '0'
     Number of samples to collect per million spans for distributed tracing. Disabled by default.
 
-v2 Proxy (to be deprecated in v3.6):
+v2 Proxy (Deprecated and to be decommissioned in v3.6):
```

Comment:
v3.7?

Reply:
The flags have already been decommissioned/removed in 3.6/main branch.

Reply:
Ok, so this PR is about making the warning more visible and standardized between branches. Thanks!